lib/src/impl_js/impl_js.utils.dart

```diff
@@ -99,6 +99,29 @@ Object _translateDomException(
       '"${e.name}", message: $message');
 }
 
+/// Convert [Error] to [UnknownError].
+/// dart2wasm throws _JavaScriptError, but _JavaScriptError is not exposed.
+///
+/// [1]: https://github.com/dart-lang/sdk/issues/55496
+/// [2]: https://api.dart.dev/stable/latest/dart-core/Error-class.html
+Object _translateJavaScriptException(Error e) {
+  final errorMessage = e.toString();
+  final message = 'browser threw "$errorMessage"';
```

Comment:
Is `errorMessage` not always going to be `"JavaScriptError"` ?

Reply:
The UnknownError message can also be a constant, so it has been fixed. [4e8e9de](https://github.com/google/webcrypto.dart/pull/178/commits/4e8e9de977dd1b6dddde98c20c526c3d59623ce1)